modules/src/core/ics26_routing/context.rs

```diff
@@ -31,9 +44,12 @@ pub trait Ics26Context:
     + ConnectionKeeper
     + ChannelKeeper
     + ChannelReader
-    + PortReader
+    + ChannelCapabilityKeeper<CoreModuleId, Capability = <Self as Ics26Context>::Capability>
```

Comment:
All `Capability` related keepers that the `Ics26Context` depends on, are scoped to the `CoreModule` and all associated `Capability` types are bound to be the same.